### PR TITLE
Add a faded background to the tab arrow

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add a faded background to the tab arrow. 
Made the placement of the arrow more logical.

## QA
- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/tabs/
- Shrink the screen till the arrow goes over the text
- See there is a white gradient behind the arrow

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1900

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review
- @anthonydillon design looks good on Demo 👍 
- But on Percy visual diffs vertical alignment is weirdly off...

screen shot 2018-07-24 at 14 29 43
screen shot 2018-07-24 at 14 29 54

**Comment by anthonydillon** [comment]
@kwm14 looking ok here on both FF and chrome. I think it might be Percy :confused:

File: scss/_patterns_tabs.scss [hunk 1]
```diff
@@ -8,18 +8,19 @@
...
 
     &::before {
-      bottom: 0;
+      background: linear-gradient(to right, $color-transparent 0%, $color-x-light 45%, $color-x-light 100%);
       color: $color-mid-dark;
       content: '\203A';
       display: block;
       font-size: 2rem;
-      line-height: map-get($line-heights, default-text);
+      line-height: calc(100% + 1rem - 3px);
```
Comment (Caleb-Ellis) [inline scss/_patterns_tabs.scss]: Can you replace `3px` with `$bar-thickness`?

**Comment by Caleb-Ellis** [comment]
@anthonydillon @kwm14 If I remember right, Percy uses an old-ish version of Firefox to take the snapshots. Anyway it looks good on Firefox 52 (which we [support](https://github.com/canonical-webteam/practices/blob/master/coding/browser-support.md)), and like the Percy screenshot on Firefox 45 (something to do with using calc in a pseudoselector).

**Pull request merged**